org.eclipse.xtext.ui.tests/src-gen/org/eclipse/xtext/ui/tests/editor/contentassist/services/GH341TestLanguageGrammarAccess.java

```diff
@@ -274,7 +274,7 @@
 		private final ParserRule rule = (ParserRule) GrammarUtil.findRuleForName(getGrammar(), "org.eclipse.xtext.ui.tests.editor.contentassist.GH341TestLanguage.STREAM");
 		private final Keyword cStreamKeyword = (Keyword)rule.eContents().get(0);
 		
-		//fragment STREAM *:
+		//fragment STREAM*:
```

Comment:
@ArneDeutsch @cdietrich I'm not sure about that one. For wildcard fragments, I've a slight preference towards a space before the `*` wildcard. What do you think?

Reply:
Would not agree here ... looks ok for me.